frontend/src/packages/auth/auth-api.ts

```diff
@@ -60,6 +60,23 @@ class AuthApi extends HttpApi {
 
     return await response.json<UserSignInResponseDto>();
   }
+
+  public async getCurrentUser(): Promise<
+    CustomerSignUpResponseDto | BusinessSignUpResponseDto
+  > {
+    const response = await this.load(
+      this.getFullEndpoint(AuthApiPath.CURRENT, {}),
+      {
+        method: 'GET',
+        contentType: ContentType.JSON,
+        hasAuth: true,
```

Comment:
this hasAuth means, that localstorage is expected to have jwt token from the previous session.
Check the behavior by clearing the localstorage or by opening the app in private tab of the browser.